packages/commonwealth/client/scripts/views/pages/discussions/CommentTree/CommentTree.tsx

```diff
@@ -485,6 +496,7 @@ export const CommentTree = ({
                   comment={comment}
                 />
               </div>
+              <div ref={scrollToRef}></div>
```

Comment:
lets wrap `<CreateComment/>` inside this, ex:
```
<div ref={scrollToRef}>
  <CreateComment
    handleIsReplying={handleIsReplying}
    parentCommentId={parentCommentId}
    rootThread={thread}
    canComment={canComment}
  />
</div>
```

and modify styles accordingly.

Reply:
It's probably some CSS hiding it. The `scrollToRef.current` will get the ref, once `CreateComment` mounts. This is just some layout organization, not a blocker, and can also be skipped. I will leave to you.